catalog/core/catalog-core-standardframework/src/test/groovy/ddf/catalog/impl/operations/ResourceOperationsSpec.groovy

```diff
@@ -192,4 +192,11 @@ class ResourceOperationsSpec extends Specification {
         policyResponse.operationPolicy() >> [:]
         return policyResponse
     }
+
+    FederatedSource mockFederatedSource(def id) {
```

Comment:
✏️ These mocks can be inlined instead. I don't think this helper method provides much value:
```groovy
federatedSources = [Mock(FederatedSource) {
    getId() >> 'resourcename'
}]
```